src/sb_utils.R

```diff
@@ -1,4 +1,12 @@
-sb_replace_files <- function(sb_id, ..., file_hash){
+
+#' @param use_task_table logical specifying whether to call `do_item_replace_tasks`
+#' which will create a task_table of the files to push to ScienceBase. This prevents them
+#' all from failing if one fails.
+#' @param task_yml only needed if `use_task_table = TRUE`; task makefile name
+#' @param out_dir only needed if `use_task_table = TRUE`; path to the directory where 
+#' final target indicators from the task_makefile should go
+#' 
+sb_replace_files <- function(sb_id, ..., file_hash, use_task_table = FALSE, task_yml, out_dir){
```

Comment:
What about updating the function to this: `sb_replace_files(sb_id, ..., file_hash, use_task_table = TRUE)`? 

I think the task table should be TRUE by default because it is more robust to upload failures. I am also suggesting below that you remove the additional arguments if possible.